components/webdriver_server/lib.rs

```diff
@@ -714,22 +714,29 @@ impl Handler {
         )))
     }
 
-    fn browsing_context_script_command(
+    fn browsing_context_script_command<const CHECK_OPEN: bool>(
         &self,
         cmd_msg: WebDriverScriptCommand,
     ) -> WebDriverResult<()> {
         let browsing_context_id = self.session()?.browsing_context_id;
-        self.verify_browsing_context_is_open(browsing_context_id)?;
+        if CHECK_OPEN {
+            self.verify_browsing_context_is_open(browsing_context_id)?;
+        }
         let msg = EmbedderToConstellationMessage::WebDriverCommand(
             WebDriverCommandMsg::ScriptCommand(browsing_context_id, cmd_msg),
         );
         self.constellation_chan.send(msg).unwrap();
         Ok(())
     }
 
-    fn top_level_script_command(&self, cmd_msg: WebDriverScriptCommand) -> WebDriverResult<()> {
+    fn top_level_script_command<const CHECK_OPEN: bool>(
+        &self,
+        cmd_msg: WebDriverScriptCommand,
+    ) -> WebDriverResult<()> {
```

Comment:
Couldn't we just pass a parameter here? The compile time gain from const generic seems very minimum here.